src/components/article-card/article-card.jsx

```diff
@@ -1,32 +1,78 @@
 import "./article-card.css";
 import { useNavigate } from "react-router-dom";
+import * as api from '../../utils/api'
+import axios from "axios";
+import { useState } from "react";
 
 
-const ArticleCard = ({ article }) => {
+const ArticleCard = ({ article, setArticlesList }) => {
+
+  const [selectedArticle, setSelectedArticle] = useState(article)
+  const [disableAdd, setDisableAdd] = useState(false)
+  const [disableRemove, setDisableRemove] = useState(false)
+
+  const addVote = () => {
+    axios
+    .patch(
+      `https://sizens-nc-news-app.herokuapp.com/api/articles/${article.article_id}`,
+      {
+        inc_votes: 1
+      }
+    )
+    .then((res) => console.log(res))
+    .catch((err) => console.log(err, "Something went wrong, cannot add vote!"))
+
+    setSelectedArticle((currentSelectedArticle) => {
+        return {...currentSelectedArticle, votes: currentSelectedArticle.votes + 1}
+      })
+      setDisableAdd(true)
+  }
+
+  const removeVote = () => {
+    axios

```

Comment:
I also don't understand why this is here. You can use one api call to amend the votes by number sent. Exactly how you have it in the back end. Adding 1 or adding -1 will achieve the upvote and downvote.